jest.setup.js

```diff
@@ -1,5 +1,5 @@
 import '@testing-library/jest-dom';
-import './setupTests';
+// import './setupTests';
```

Comment:
why comment?

Reply:
@vinit717 tests is not setup on members-site

Reply:
@vinit717 resolved. Will fix this in different pr.

Reply:
Can you mention the issue ticket for test fix

Reply:
I have made it draft for now, some test are failing.

Reply:
https://github.com/Real-Dev-Squad/members-site/pull/170